[2024-10-18T13:37:56.047+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-18T13:37:56.084+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_1st_task scheduled__2024-09-26T00:00:00+00:00 [queued]>
[2024-10-18T13:37:56.106+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_1st_task scheduled__2024-09-26T00:00:00+00:00 [queued]>
[2024-10-18T13:37:56.107+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-18T13:37:56.123+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): this_is_1st_task> on 2024-09-26 00:00:00+00:00
[2024-10-18T13:37:56.130+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19318) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-18T13:37:56.131+0000] {standard_task_runner.py:72} INFO - Started process 19322 to run task
[2024-10-18T13:37:56.131+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'this_is_1st_task', 'scheduled__2024-09-26T00:00:00+00:00', '--job-id', '5970', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp9yvse_7g']
[2024-10-18T13:37:56.134+0000] {standard_task_runner.py:105} INFO - Job 5970: Subtask this_is_1st_task
[2024-10-18T13:37:56.210+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v4.this_is_1st_task scheduled__2024-09-26T00:00:00+00:00 [running]> on host ad4831ffc3d7
[2024-10-18T13:37:56.341+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sourabh' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='this_is_1st_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-26T00:00:00+00:00'
[2024-10-18T13:37:56.343+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-18T13:37:56.370+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-18T13:37:56.371+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is my first task']
[2024-10-18T13:37:56.387+0000] {subprocess.py:86} INFO - Output:
[2024-10-18T13:37:56.400+0000] {subprocess.py:93} INFO - Hello world, this is my first task
[2024-10-18T13:37:56.402+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-18T13:37:56.436+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-18T13:37:56.437+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=this_is_1st_task, run_id=scheduled__2024-09-26T00:00:00+00:00, execution_date=20240926T000000, start_date=20241018T133756, end_date=20241018T133756
[2024-10-18T13:37:56.567+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-18T13:37:56.618+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
